packages/nextra/src/mdx-plugins/remark.ts

```diff
@@ -32,7 +30,7 @@ export function getFlattenedValue(node: Parent): string {
     .join('')
 }
 
-export default function remarkHeadings(this: Processor) {
+export function remarkHeadings(this: Processor) {
```

Comment:
it's better to use named export, as in `compile.ts` this function is imported as `remarkHandler`